### PR TITLE
fixed  vmselect args

### PR DESCRIPTION
**Pull request by zhiyin009**
- Fixed field in vmagent example
- Fixed vmselect args logic

### CODE REVIEW

File: controllers/factory/vmcluster.go [hunk 1]
```diff
@@ -527,8 +527,8 @@ func makePodSpecForVMSelect(cr *v1beta1.VMCluster, c *config.BaseOperatorConf) (
 	}
 	if cr.Spec.ReplicationFactor != nil {
 		var dedupIsSet bool
-		for _, arg := range cr.Spec.VMSelect.ExtraArgs {
-			if strings.Contains(arg, "-dedup.minScrapeInterval") {
+		for arg, _ := range cr.Spec.VMSelect.ExtraArgs {
```
Comment (f41gh7) [inline controllers/factory/vmcluster.go]:
```suggestion
		for arg := range cr.Spec.VMSelect.ExtraArgs {
```
 I suggest to change signature, in this case value can be omitted.

**Comment by f41gh7** [comment]
Thanks! Looks good.

**Comment by f41gh7** [comment]
Thanks for contribution!

**Comment by zhiyin009** [comment]
😊

**Pull request merged**